making-demos.md

```diff
@@ -171,6 +171,27 @@ TODO
 
 TODO
 
+### Prepending the Firefox DevTools intro video
+
+It's a good idea to also prepend the Firefox DevTools intro video in your new video so it looks more official.
+
+You can download this video [on GitHub's LFS storage here](https://github.com/devtools-html/demos/blob/master/assets/bumper%201080.mov).
```

Comment:
You're right, it's not relevant. The file will be shown as normal on GitHub.com and will also be downloaded as part of a `git clone`, so I'll remove the mention about LFS.